### PR TITLE
Fix/test suite

### PR DESCRIPTION
**Pull request by DimCitus**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,6 +6,15 @@ TEST_CONTAINER_NAME = pg_auto_failover-test
 DOCKER_RUN_OPTS = --cap-add=SYS_ADMIN --cap-add=NET_ADMIN -ti --rm
 PDF = ./docs/_build/latex/pg_auto_failover.pdf
 
+# TEST indicates the testfile to run
```
Comment (mtuncer) [inline Makefile]: Like the feature. wonder if using lowercase name would be nicer

Comment (DimCitus) [reply Makefile]: The way make works, the variable can be set in the environment or just for the make command. That's why I believe it's usually uppercase.

```
$ TEST=test_auth make run-test
```

File: tests/pgautofailover_utils.py [hunk 2]
```diff
@@ -473,7 +482,8 @@ def set_replication_quorum(self, replicationQuorum):
             Sets replication quorum via pg_autoctl
         """
 
-        set_command = [shutil.which('pg_autoctl'), 'set', 'node', '--pgdata', self.datadir,
+        set_command = [shutil.which('pg_autoctl'),
```
Comment (mtuncer) [inline tests/pgautofailover_utils.py]: can we hava PGAutoCtl class handle this too ?

File: tests/pgautofailover_utils.py [hunk 3]
```diff
@@ -657,14 +672,104 @@ def failover(self, formation='default', group=0):
...
+
+    def communicate(self):
+        """
+        Read all data from the Unix PIPE
+        """
+        self.out, self.err = self.run_proc.communicate()
+
+        return self.out, self.err
+
+    def readmore(self, secs=1):
```
Comment (mtuncer) [inline tests/pgautofailover_utils.py]: can you re-think the name

File: tests/test_create_run.py [hunk 4]
```diff
@@ -75,19 +75,3 @@ def test_010_drop_secondary():
...
     assert node2.wait_until_state(target_state="single")
-    
-def test_011_add_new_secondary():
-    global node3
-    node3 = cluster.create_datanode("/tmp/basic/node3")
-    node3.create(run = True)
-    assert node3.wait_until_state(target_state="secondary")
-    assert node2.wait_until_state(target_state="primary")
-
-def test_012_multiple_manual_failover():
```
Comment (mtuncer) [inline tests/test_create_run.py]: any reason why we don't need this anymore ?

File: tests/pgautofailover_utils.py (hunk 3 above)
Comment (DimCitus) [reply tests/pgautofailover_utils.py]: done, thanks!

File: tests/pgautofailover_utils.py (hunk 2 above)
Comment (DimCitus) [reply tests/pgautofailover_utils.py]: Good point, I did that in the follow-up commit, now pushed.

File: tests/test_create_run.py (hunk 4 above)
Comment (DimCitus) [reply tests/test_create_run.py]: I don't believe those tests are expecting any specific behaviour in `create --run` and we test them already in the normal `create` then `run` cases.

**Pull request merged**